### DISCUSSION TITLE
How to completely uninstall this ui fix ?

### DISCUSSION THREAD

**Discussion by T1mL3arn** [black7375/Firefox-UI-Fix#536] [Q&A] [answered]
How do I completely uninstall this ui fix and go back to original Firefox UI ?

**Comment by black7375** [comment, marked as answer]
Remove `user.js` file, `chrome` directory at profile.

**Reply by T1mL3arn** [reply]
Is it safe to just remove the whole chrome dir since I have no other tweaks related to browser ui ?

**Reply by black7375** [reply]
Yes, if you remove the `chrome` directory, it will work with the original UI.

**Comment by Aleksandra0056** [comment]
Why you wanted to uninstall the Firefox UI Fix? What is wrong with it?

**Reply by T1mL3arn** [reply]
>Why you wanted to uninstall the Firefox UI Fix? 

I don't remember.
